src/components/dropdowns/BasicDropDown.js

```diff
@@ -2,12 +2,26 @@ import React, { Component } from 'react';
 import { Layout, Select, Text, Card, CardHeader } from '@ui-kitten/components';
```

Comment:
I don't think that this dropdown is ever used. If you want, feel free to delete the component and all its references.